src/components/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+import {ListItemColumn, Teams, UserData} from 'types/types';
+import {CardContainer} from './Card.styled';
+
+interface CardProps {
+    id: string;
+    url?: string;
+    columns: ListItemColumn[];
+    hasNavigation?: boolean;
+    navigationProps?: UserData | Teams;
+}
+
+const Card = (props: CardProps): JSX.Element => {
+    const {
+        id,
+        columns,
+        url =  '',
+        hasNavigation = true,
+        navigationProps = null,
+    } = props;
+
+    const navigate = useNavigate();
+
+    const handleOnClick = (event: React.MouseEvent<HTMLDivElement>): void => {
+        event.preventDefault();
+
+        if (hasNavigation) {
+            navigate(url, {
```

Comment:
this is the main reason I added a default `url`, because `hasNavigation` is `true` by default, but `url ` can be `undefined`